pkg/controller/instance/instance_controller.go

```diff
@@ -472,7 +482,7 @@ func startPlanExecution(i *v1beta1.Instance, planName string, ov *v1beta1.Operat
 		}
 	}
 	if notFound {
-		return &v1beta1.InstanceError{Err: fmt.Errorf("asked to execute a plan %s but no such plan found in instance %s/%s", planName, i.Namespace, i.Name), EventName: kudo.String("PlanNotFound")}
+		return &v1beta1.InstanceError{Err: fmt.Errorf("asked to execute a plan %s but no such plan found in instance %s/%s", planName, i.Namespace, i.Name), EventName: convert.String("PlanNotFound")}
```

Comment:
why would an error message need the ptr to the string instead of a string?

Reply:
Not sure. But that's how `InstanceError` is defined. Refactoring it shouldn't be part of this PR.